app/subscriber/src/features/my-minister/MyMinister.tsx

```diff
@@ -68,12 +68,13 @@ export const MyMinister: React.FC = () => {
 
   /** retrigger content fetch when change is applied */
   React.useEffect(() => {
```

Comment:
Isn't this the 3rd PR related to this?

Reply:
Correct yes, annoyingly have to throw things against the wall since I can't recreate locally :(